quality/metrics/bleu.md

```diff
@@ -0,0 +1,31 @@
+---
+grand_parent: Building and research
+parent: Metrics
+nav_order: 1
+title: BLEU
+description: BiLingual Evaluation Understudy
+---
+
+**BLEU** (BiLingual Evaluation Understudy) is the standard metric for machine translation evaluation.
+
+> The closer a machine translation is to a professional human translation, the better it is.
+
+BLEU compares machine translation output to a single reference translation.
+BLEU is an [n-gram](/../concepts/n-gram.md) word-based metric.
+
+BLEU is based on translation precision.
+Its basic unit of evaluation is the sentence.
+
+BLEU treats all n-grams equally.
+
+### Flavours
+
+- sacreBLEU
+- BLEUrt
+- M-BLEU
+
+*Note: The list is incomplete.*
```

Comment:
> Because BLEU is not an exactly defined metric, BLEU scores are not comparable.
> 
> So researchers have created *flavours* that are more concretely defined.
> 
> Here are some of the most well-known flavours of BLEU:

But I'm also fine with just having less :-D

Reply:
Maybe this belongs in the top-level metrics comment, since the idea of flavours is true for many metrics.

Maybe "flavours" is too slangy a word.  "variants"?